x/ecocredit/spec/01_concepts.md

```diff
@@ -1 +1,45 @@
 # Concepts
+
+## Credit Type
+
+A credit type is the primary indicator for the methodology defined within a credit class. A credit type includes a name (e.g. carbon, biodiversity), an abbreviation (a set of 1-3 uppercase characters), a measurement unit (e.g. kilograms, tons), and a decimal precision.
+
+The credit type abbreviation is used to construct the credit class ID. For example, "C1" is the ID for the first credit class that uses the "carbon" credit type where "C" is the credit type abbreviation and "1" is the sequence number of the credit class for the given credit type.
+
+Credit types are listed as an on-chain parameter. Adding a new credit type to the list of approved credit types requires a parameter change proposal.
+
+## Credit Class
+
+A credit class defines the class of an ecosystem service credit. A credit class includes metadata that defines a methodology for measuring and monitoring changes in ecological state. The credit class metadata defines the structure, procedures, and requirements of the methodology.
+
+Each credit class is associated with a single credit type, which is the primary indicator for an ecosystem service methodology. Each credit class includes a credit class admin (the creator and maintainer of the credit class) and a list of approved credit issuers.
+
+## Credit Class Creator
```

Comment:
```suggestion
## Credit Class Creator Allowlist
```

I think that to discuss "credit class creators" outside of the context of the allow list is a bit confusing, so maybe we should add "allowlist" here and speak about that as opposed to talking about "credit class creator" as a role in its own right?

For example, when we eventually disable the allowlist entirely, will it make much sense to talk about "credit class creator" as a distinct role, since everyone will be able to do it?

Reply:
Good idea! I see how it might cause confusing alongside the other two roles.